src/runtime/test/storage/crdt-collection-model-test.ts

```diff
@@ -77,6 +77,6 @@ describe('crdt-collection-model', () => {
     assert.sameMembers(model.getKeys('keys'), ['key1']);
     const effective = model.remove('nokeys', []);
     assert.isTrue(effective);
-    assert(model.size, 1);
+    assert.equal(model.size, 1);
```

Comment:
lol

Reply:
This was a compile error! Turns out type checking is useful sometimes :-)